src/components/AddNewUserModal.module.css

```diff
@@ -0,0 +1,33 @@
+@import '../common/global.css';
+
+/* .box {
+  width: 80vw;
+} */
+
+.profilePic {
+  background-color: white;
+  width: 4vw;
+  margin: auto;
+  margin-bottom: 3vh;
+  margin-top: 3vh;
+}
+
+.modalHeader {
+  margin: auto;
+}
+
+.buttonContainer {
+  display: flex;
+  align-items: center;
+  justify-content: center;
```

Comment:
[suggestion] make `justify-content` `space-between` since buttons have to be on opposite ends of the modal